core/types/block_ext_test.go

```diff
@@ -370,24 +373,19 @@ func TestNewBlockWithExtData(t *testing.T) {
 
 	tests := []struct {
 		name      string
-		header    func() *Header
```

Comment:
I always hate when I have to introduce a constructor into a table test so it's nice to see this disappearing :)